client/components/videos-ui/index.jsx

```diff
@@ -7,6 +7,18 @@ import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import getSelectedSiteSlug from 'calypso/state/ui/selectors/get-selected-site-slug';
 import './style.scss';
 
+const VideoPlayer = ( { videoUrl } ) => {
```

Comment:
Technically we should pull this out to its own file, being a legit standalone component (eg. `client/components/videos-ui/video-player.jsx`), but we can do that as clean up if we have time.

Reply:
Great, I'll split it in the future.